transcom/mymove
# B-17836 Add Space Force to Branch of Service

Agility ticket: [B-17836](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a863705)

## Summary

This change adds "Space Force" to the list of Branch of Services in the drop down list.

- For users, the value will be visible on the "Branch of Services" drop down list on the "Create your profile" page.

- For SC and TOO, it will be visible on the "Brach of Services" drop down list on the "View Allowances" page.

## How to test

For customer
* Start a new move
* On the "Create your profile" page, click the "Branch of Services" drop down and confirm "Space Force" is in the list.

For  SC/TOO
* Select a move in the list
* Edit Allowances
* On the "View Allowance" page, click the "Branch of Services" drop down and confirm "Space Force" is in the list. 
* Scroll to the bottom and click "Save".
* Select "Move History" tab
* Confirm the "Space Force" appears for the "Branch of Service" change.


## Screenshots
Customer
![image](https://github.com/transcom/mymove/assets/146897935/2f9180b9-948a-4f33-aa7b-b5b5b66627aa)

SC/TOO
![image](https://github.com/transcom/mymove/assets/146897935/691023b7-367f-4c86-bdcd-4adf330722a7)
![image](https://github.com/transcom/mymove/assets/146897935/bb6e6d31-6412-4a83-aa65-88b0a21c2770)
